packages/notifications/src/styled/global-alert/StyledGlobalAlert.ts

```diff
@@ -124,18 +132,24 @@ export const StyledGlobalAlert = styled.div.attrs({
   box-sizing: border-box;
   direction: ${props => (props.theme.rtl ? 'rtl' : 'ltr')};
 
+  ${sizeStyles}
+  ${colorStyles}
+
   && a {
     border-radius: ${props => props.theme.borderRadii.sm};
     text-decoration: underline;
 
-    &:focus {
+    /* [1] */
+    ${SELECTOR_FOCUS_VISIBLE} {
+      text-decoration: none;
+    }
+
+    /* [1] */
+    &:hover {
       text-decoration: underline;
```

Comment:
If design intent is _not_ to have the underline on focus, then it should not be reapplied on focus+hover.

Reply:
Oof, good catch.

Reply:
This treatment appears to be fixed, but the problem still exists on `<Anchor>` in the buttons package.

Reply:
...or is `<Anchor>` focus+hover working as intended by design, @lucijanblagonic?

Reply:
I would keep the underline on focus, to be consistent with how `Anchor` work.